### PR TITLE
Update alerts.blade.php

### PR DESCRIPTION
**Pull request by pxpm**

### CODE REVIEW

**Review by promatik — APPROVED** [review]
Tested and it's working

**Comment by tabacitu** [comment]
Note: we should merge this AFTER we merge https://github.com/Laravel-Backpack/CRUD/pull/3079

**Review by promatik — APPROVED** [review]

**Pull request merged**
